support-frontend/assets/helpers/render.js

```diff
@@ -19,10 +19,10 @@ const getElementOrBody = (id: ?string): HTMLElement => {
   return element;
 };
 
-const renderError = (id: ?string) => {
+const renderError = (e: any, id: ?string) => {
   const element = getElementOrBody(id);
 
-  logException(['Fatal error rendering a page', id ? `id:${id}` : null].join(' '));
+  logException(['Fatal error rendering a page.', id ? `id:${id}.` : null, 'Error string:', e].join(' '));
```

Comment:
not your changes, but can we just use string interpolation here?

Reply:
sure thing